scripts/release_scripts/run_macaron.sh

```diff
@@ -439,6 +443,16 @@ if [[ -n "${arg_prov_exp:-}" ]]; then
     fi
 fi
 
+# Determine the provenance expectation path to be mounted into ${MACARON_WORKSPACE}/prov_files/${pf_name} where pf_name is a file name.
```

Comment:
```suggestion
# Mount the provenance file into ${MACARON_WORKSPACE}/prov_files/${pf_name} where pf_name is a file name.
```

Reply:
Fixed in https://github.com/oracle/macaron/pull/698/commits/bf2f4078e4d4c6f85e29ffb1dd7cb962b5f0ec6c